### PR TITLE
MAKE-796: fix windows

### PR DESCRIPTION
**Pull request by julianedwards**

### CODE REVIEW

File: evergreen.yaml [hunk 1]
```diff
@@ -23,18 +24,33 @@ variables:
...
 functions:
+  git-config-windows-api-length:
+    command: shell.exec
+    type: setup
+    params:
+      silent: true
+      script: |
+        if [ "Windows_NT" == "$OS" ]; then
+          git config --system core.longpaths true
+        fi
```
Comment (tychoish) [inline evergreen.yaml]: can we do this in the makefile and leave out the --system

File: evergreen.yaml [hunk 2]
```diff
@@ -23,18 +24,33 @@ variables:
...
     params:
       silent: true
       script: |
-        mkdir ~/.aws
-        cat <<EOF > ~/.aws/config
+        if [ "Windows_NT" == "$OS" ]; then
+          export AWS_DIR=/cygdrive/c/Users/$USER/.aws
+        else
+          export AWS_DIR=$HOME/.aws
+        fi
```
Comment (tychoish) [inline evergreen.yaml]: push to makefile?

Comment (julianedwards) [reply evergreen.yaml]: don't want people to wipe out their aws directories by accident

File: evergreen.yaml (hunk 1 above)
Comment (julianedwards) [reply evergreen.yaml]: this is temporary anyway, I am going to put a TODO now with the build ticket number

**Pull request merged**